vscode_extension/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "AaC",
+    "name": "aac",
```

Comment:
I changed it because I was seeing a warning for this when it had capitalized letters:

```
String does not match the pattern of "^(?:@[a-z0-9-*~][a-z0-9-*._~]*/)?[a-z0-9-~][a-z0-9-._~]*$".
```

Reply:
Sounds good. Thanks!